lib/addMethod/globalize/afterFailure.js

```diff
@@ -13,9 +13,10 @@ module.exports = function (config, body, params) {
 
     // Run global promise first
     .then(function () {
-      if (_.isFunction(threadneedle._globalOptions.afterFailure) && config.globals !== false) {
-        return when(threadneedle._globalOptions.afterFailure(body, params));
-      }
+        var localOnly = config.globals === false || _.get(config, 'globals.afterFailure', true) === false;
```

Comment:
This code is repeated quite a lot, perhaps it could be moved to a helper function so you can do

```
helper.isDisabled(config, 'afterFailure')
```